templates/home.hbs

```diff
@@ -94,24 +94,23 @@ read_pages_glob = ["events/*.md"]
                     </div>
 
                     <div class="column is-one-third-desktop is-full-touch">
-                        <a class="card-link" href="/bartholomew/index" target="_self">
+                        <a class="card-link" href="/wasm-languages/webassembly-language-support" target="_self">
                             <div class="card card-one project">
                                 <div class="card-image dark">
                                     <figure class="image">
-                                        <img src="{{site.info.base_url}}/static/image/project-left-bart-dark.png"
-                                            alt="Bartholemew" />
+                                        <img src="{{site.info.base_url}}/static/image/project-left-wasm-dark.png"
+                                            alt="WasmLanguages" />
                                     </figure>
                                 </div>
                                 <div class="card-image light">
                                     <figure class="image">
-                                        <img src="{{site.info.base_url}}/static/image/project-left-bart-light.png"
-                                            alt="Bartholemew" />
+                                        <img src="{{site.info.base_url}}/static/image/project-left-wasm-light.png"
+                                            alt="WasmLanguages" />
                                     </figure>
                                 </div>
                                 <div class="card-content">
-                                    <h3><span>A micro-CMS for WebAssembly &amp;
-                                            Spin</span></h3>
-                                    <p>Compiled to WebAssembly, Bartholomew can run in any Spin environment.</p>
+                                    <h3><span>WebAssembly Language Support Matrix</span></h3>
+                                    <p>This guide tracks support for compiling a language to WebAssembly.</p>
```

Comment:
A non-essential idea: the other entries on the landing page are more call to action like.  I wonder if something like "How ready for WebAssembly is your favourite language?  Find out here." or something like that would be more in keeping.  We can discuss later with punchier wordsmiths though - the text you have here is fine!